CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
+- Removes all references to flowdock and new relic third party integrations [\#1616](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1616) ([oarbusi](https://github.com/oarbusi))
+- Fixes documentation errors in cloud provider access [\#1615](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1615) ([milh0use](https://github.com/milh0use))
+- Fixes search index acceptance tests flow [\#1610](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1610) ([lantoli](https://github.com/lantoli))
+- Bumps github.com/aws/aws-sdk-go from 1.47.4 to 1.47.5 [\#1608](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1608) ([dependabot[bot]](https://github.com/apps/dependabot))
+- Bumps github.com/gruntwork-io/terratest from 0.46.1 to 0.46.6 [\#1607](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1607) ([dependabot[bot]](https://github.com/apps/dependabot))
+- Bumps github.com/aws/aws-sdk-go from 1.46.3 to 1.47.4 [\#1606](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1606) ([dependabot[bot]](https://github.com/apps/dependabot))
+- Adds type and fields attributes in resource and data sources for `search_index` [\#1605](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1605) ([lantoli](https://github.com/lantoli))
+- Updates Atlas Go SDK [\#1604](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1604) ([lantoli](https://github.com/lantoli))
+- Disables `event_trigger` from acceptance test due to missing cluster. [\#1603](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1603) ([marcosuma](https://github.com/marcosuma))
+- Reduces stale days to 5 and close after 2 day of stale [\#1602](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1602) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- Fixes Update SDK GitHub action [\#1596](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1596) ([lantoli](https://github.com/lantoli))
+- Disables `data_source` `event_trigger` tests since they are failing [\#1595](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1595) ([marcosuma](https://github.com/marcosuma))
+- Changes naming convention for `data_source` event trigger test. [\#1594](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1594) ([marcosuma](https://github.com/marcosuma))
+- Changes naming convention for `data_source` event trigger tests. [\#1593](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1593) ([marcosuma](https://github.com/marcosuma))
+- Updates RELEASING.md [\#1592](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1592) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- Reverts "chore: add acceptance tests and migration tests to report in merge queue" [\#1591](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1591) ([andreaangiolillo](https://github.com/andreaangiolillo))
```

Comment:
Should we include reverts in the CHANGELOG or should we remove this line and the next since they cancel each other?

Reply:
I am in favour of removing

Reply:
Since we noticed it we can remote, but overall I don't think it's that critical

Reply:
if both the original and remote are in this release, they annihilate each other and we can just remove them both :-)

Reply:
Removed